test/org/sagebionetworks/bridge/dynamodb/DynamoTaskDaoMockTest.java

```diff
@@ -120,8 +123,8 @@ public void testOfFirstPeriod() throws Exception {
         assertTask("2015-04-12T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(0));
         assertTask("2015-04-13T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(1));
         assertTask("tapTest", TestConstants.ACTIVITY_3_REF, tasks2.get(2));
-        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(3));
-        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_1_REF, tasks2.get(4));
+        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_1_REF, tasks2.get(3));
+        assertTask("2015-04-14T13:00:00-07:00", TestConstants.ACTIVITY_2_REF, tasks2.get(4));
```

Comment:
It's not immediately clear from the test why the sort order is the way it is. For example, why is tapTest seemingly randomly in the middle? What determines the sort order between activity 1 and activity 2?